### PR TITLE
Fix for --without-pull not working in install.sh

### PR DESCRIPTION
**Pull request by Kruat**
- [X] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [ ] I wrote some lines in the [book](https://github.com/radareorg/radare2book) (optional)

**Description**

### CODE REVIEW

File: sys/install.sh [hunk 1]
```diff
@@ -32,6 +32,11 @@ while : ; do
 		shift
 		continue
 		;;
+	"--without-pull")
+		export WITHOUT_PULL=1
+		shift
```
Comment (trufae) [inline sys/install.sh:37]:
```suggestion
```

File: sys/install.sh [hunk 2]
```diff
@@ -32,6 +32,11 @@ while : ; do
...
+		continue
```
Comment (trufae) [inline sys/install.sh:38]:
```suggestion
```

File: sys/install.sh [hunk 3]
```diff
@@ -62,7 +67,7 @@ cd "$(dirname $0)"/..
 pwd
 
 # update
-if [ "$1" != "--without-pull" ]; then
+if [ -z $WITHOUT_PULL ]; then
```
Comment (trufae) [inline sys/install.sh:70]:
```suggestion
if [ -z "$WITHOUT_PULL" ]; then
```

**Review by trufae — CHANGES_REQUESTED** [review]

File: sys/install.sh (hunk 1 above)
Comment (Kruat) [reply sys/install.sh]: why? the original code shifts it away. Do you want to pass it on thru ARGS?

File: sys/install.sh (hunk 2 above)
Comment (Kruat) [reply sys/install.sh]: the continue is important if we do shift, to avoid edge cases.

Comment (trufae) [reply sys/install.sh]: After the case theres a shift and the continue is implicit because it loops again

File: sys/install.sh (hunk 1 above)
Comment (trufae) [reply sys/install.sh]: Uhm maybe i missread the patch :? Ill check after lunch

File: sys/install.sh (hunk 2 above)
Comment (Kruat) [reply sys/install.sh]: yeah, sorry, you're in the right.

**Pull request merged**